aura/lib/Aura/Languages.hs

```diff
@@ -917,14 +967,16 @@ cleanStates_5 t = \case
 cleanStates_6 :: Int -> Language -> Doc AnsiStyle
 cleanStates_6 n = \case
   Polish    -> pretty n <+> "jest przypiętych i nie zostanie usuniętych."
+  Arabic    -> ".اذا كانو مثبتين ولا يمكن ازالتهم " <+> pretty n
   Spanish   -> pretty n <+> "de estos están anclados y no se eliminarán."
   Italian   -> pretty n <+> "di questi sono stati fissati, perciò non saranno rimossi."
   Dutch     -> pretty n <+> "hiervan zijn vastgezet, en worden niet verwijderd."
-  _         -> pretty n <+> "of these are pinned, and won't be removed."
```

Comment:
why did you change that?

Reply:
It's a reference to this issue https://github.com/fosskers/aura/issues/725

But the original wording was correct and intended. Please do revert this change.